website/blog/2023-02-28-ferretdb-release-0-9-2.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "FerretDB v0.9.2 - Tigris Authentication support"
+slug: ferretdb-v-0-9-2
+authors: [alex]
+description: "We are delighted to announce the release of FerretDB version 0.9.2, which includes Tigris authentation, support for `$listIndexes`, `$addtoSet`, and more."
+image: /img/blog/ferretdb_v0.9.2.jpg
+tags: [release]
+date: 2023-02-28
+---
+
+We are delighted to announce the release of FerretDB version 0.9.2, which includes Tigris authentation, support for `$listIndexes`, `$addtoSet`, and more.
+
+![FerretDB v0.9.1 - Minor Release](/img/blog/ferretdb_v0.9.2.jpg)
+
+<!--truncate-->
+
+It’s been an intense and thrilling process toward bringing you the open-source alternative to FerretDB.
+
+Before we go into all the exciting new features in this release, let’s recap FerretDB and what we’re striving to achieve.
+[FerretDB](https://www.ferretdb.io/) is an [open-source alternative to MongoDB](https://blog.ferretdb.io/5-database-alternatives-mongodb-2023/) that converts MongoDB drivers and protocols into SQL with a backend on PostgreSQL.
+
+While our aim is not to be compatible with all the features in MongoDB, we hope to provide all the necessary features you need for real-life use cases, and so far, we’ve added some pretty cool features.
```

Comment:
We are gradually increasing compatibility with MongoDB, so FerretDB can act as a fully open source drop-in replacement for it. MongoDB compatibility includes tools, drivers and the entire MongoDB ecosystem. Be sure to try it out!